ui/components/FilterSummary/index.js

```diff
@@ -24,11 +24,17 @@ export function FilterSummary({ schema }) {
     updateQuery(newQuery);
   }
 
-  const activeFilters = omit(query, 'q', 'page', 'sort');
+  const chosenFilters = omit(query, 'q', 'page', 'sort');
 
-  if (!size(activeFilters)) {
+  if (!size(chosenFilters)) {
     return null;
   }
+  const { standard_category } = chosenFilters;
+  // trick to resort category to be last,
+  // TODO: better sorting needed
+  const activeFilters = standard_category
+    ? { ...chosenFilters, standard_category }
+    : chosenFilters;
```

Comment:
Rather than forcing the ordering so that category is at the end (but leaving care setting and topic potentially misordered), it feels probably better to just iterate over a fixed set of keys like we do (indirectly) [here](https://github.com/nhsx/standards-registry/blob/master/ui/components/Filters/index.js#L66).

Reply:
Agree, leaving a TODO in my branch to pick this up after merge->deploy->demo etc